### PR TITLE
Added explicit function return types

### PR DESCRIPTION
**Pull request by RohanTalip**
Please:
- [x] Make your pull request atomic, fixing one issue at a time unless there are many relevant issues that cannot be decoupled.
- [ ] Provide a test case & update the documentation in the `Readme.md`

### CODE REVIEW

**Comment by domoritz** [comment]
Why does this change improve the code?

**Comment by RohanTalip** [comment]
It helps me at least, as someone new to this code, to see at a glance what type(s) each function is returning, without relying on my IDE (VS Code) to show me that and also without needing to scroll down and find all return statements.

If you think it doesn't add much, feel free to close the pull request.

File: typescript-json-schema.ts [hunk 1]
```diff
@@ -570,7 +570,7 @@ export class JsonSchemaGenerator {
         return undefined;
     }
 
-    private getDefinitionForProperty(prop: ts.Symbol, node: ts.Node) {
+    private getDefinitionForProperty(prop: ts.Symbol, node: ts.Node): Definition|null {
```
Comment (domoritz) [inline typescript-json-schema.ts:573]:
```suggestion
    private getDefinitionForProperty(prop: ts.Symbol, node: ts.Node): Definition | null {
```

**Review by domoritz — CHANGES_REQUESTED** [review]

**Comment by domoritz** [comment]
I think it's helpful. Can you find out why Travis tests are not running?

@YousefED did you disable Travis recently?

File: typescript-json-schema.ts (hunk 1 above)
Comment (RohanTalip) [reply typescript-json-schema.ts]: > I think it's helpful.

Great!

> Can you find out why Travis tests are not running?

I'm not familiar with the Travis setup yet ...

By the way, would using a code formatter like Prettier be useful?  (I see that https://github.com/vega/ts-json-schema-generator also uses Prettier)

**Comment by RohanTalip** [comment]
> I think it's helpful.

Great!

> Can you find out why Travis tests are not running?

I'm not familiar with the Travis setup yet ...

By the way, would using a code formatter like Prettier be useful?  (I see that https://github.com/vega/ts-json-schema-generator also uses Prettier)

File: typescript-json-schema.ts (hunk 1 above)
Comment (RohanTalip) [reply typescript-json-schema.ts]: FYI, I fixed this by pushing another commit before I realised that I could just click on the `Commit suggestion` button.

**Comment by domoritz** [comment]
Yes, it would be if we can enforce it. Maybe switching to github actions would be a good idea (vega/ts-json-schema-generator uses it as well).

**Comment by YousefED** [comment]
> I think it's helpful. Can you find out why Travis tests are not running?
> 
> @YousefED did you disable Travis recently?

Didn't change anything. Maybe Travis had a hiccup as I think it's running (but failing) now?

**Pull request merged**